Another issue killer

HollishKid opened the pull request:
Avancement pré-V1.

HollishKid committed:
MAJ de PEX & corrections mineures

- Synchronisation de PEX avec discord, site et ChestCmds.
- Correctifs dans journal et donjon.

HollishKid committed:
Check d'inventaire pour MM Items et corrections

- Resolves #48
- Ajoute et modifie des permissions PEX
- Modifie la commande "mm i" en "mm items" (à utiliser exclusivement)
- Répare l'event d'xp doublé du mercredi.
- Amorce l'utilisation de SKUtilities 0.8.9

HollishKid committed:
New Color Codes & Chat Management

- Modifie les codes couleurs BQ et PEX.
- Remplace le chat manager de Towny par Skript.
- Début de création de chanels.
- Début du transfert vers discord par Skript.

HollishKid committed:
Chanels de Discussion & DiscordSkript

- Jimbo ne répond plus aux non-citoyens.
- Non-citoyens ne peuvent plus sortir de la ville.
- Réparation de bartend.
- Mise en forme de ChestCommands.
- Ajout des Chanels (commandes, menus).
- Suppression de DiscordMC et remplacement par Skript.

HollishKid committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Hollish

HollishKid committed:
Bug Fixes

- Resolves #50
- Répare Tano
- Répare prix air_wand
- Améliore et répare les chanels

HollishKid committed:
Synchronisation

- Uploads Zedd
- Uploads some Mysteria stuff
- Repairs custom_recipies

HollishKid committed:
Résolution et suite

- Resolves #53
- Continues Earth Temple

HollishKid committed:
Web Update

- Nouvelles adresses web.
- Importation de AS

HollishKid committed:
Avancement vers pré-V1

Multiples corrections, ajouts, nettoyages pour pré-V1.

HollishKid committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Hollish

# Conflicts:
#	BetonQuest/drisgan/events.yml

HollishKid committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Hollish

HollishKid committed:
Pré-V1 continued

- correction de premiers bugs sur pré-V1
- progression sur pré-V1

HollishKid committed:
WorldBoss

Ajout du Kalphite au Village des Premiers Hommes.

crashoz reviewed (approved):
Ca en fait du changement à lire, mais je ne vois pas d'erreurs énormes. PR acceptée 👍 

Je vois qu'il y a de nouvelles commandes pour les modos? Il faudrait qu'on se tienne à jour une liste des commandes peut etre.

(et je te conseille de faire des fonctions en skript parce que tu répètes beaucoup de code, ça permettrait de factoriser un peu)

HollishKid committed:
Progression pré-V1

- Ajout des Cagnottes de Métier, partage d'xp au sein d'un même corps de
métier.
- Ajout du locale de mcMMO (n'appartient PAS dans ce dossier)
